### ISSUE TITLE
Forms are not sending Notification e-mails

### ISSUE THREAD

**Issue by mollypwest** [tlovett1/custom-contact-forms#182]
Hello.
I've set up the Custom Contact Forms plug in for a website I'm working on.  The submissions are showing up in my dashboard, but I am not receiving e-mails.  Can you please assist?
Thank you.

**Comment by tlovett1** [comment]
Are you receiving WordPress lost password emails?

**Comment by mollypwest** [comment]
Thank you for the follow up—I’ve fixed the problem.

**Comment by jasonsan3** [comment]
How did you fix the problem? @mollypwest

**Comment by jasonsan3** [comment]
I managed to fix the problem on my end. This is going to be a case-by-case solution, but I thought I'd post my solution in case anyone else has similar issues.

If you have access to change your DNS records, you'll have to make sure that your MX Records are set correctly.

I have most of my emails set to a remote MX Record from my domain host to google. I do this so that I can use a customized domain with Gmail easily rather than having to set up email forwarding per user, I can just reroute all emails to google in the event that I add users, etc.

Because I set this up, my contact form submissions were being sent to the domain host rather than routed through to google. So all I had to do (after setting up the applicable TXT records to reroute all messages through my host IP), is make sure that MX Records were changed from "auto" to "remote" for all messages. 

Let me put it this way: Any emails sent via this particular contact form is considered by the host to be a remote source email rather than local. So if you are having issues with emails being sent correctly to your email, you'll have to grant permissions to remote sources in order for those messages to show up correctly to your email.

Again, this will depend on your hosting structure.
